ports/mimxrt/boards/ADAFRUIT_METRO_M7/mpconfigboard.h

```diff
@@ -58,15 +58,39 @@
 #define MICROPY_HW_WIFI_SPI_ID       (0)
 #define MICROPY_HW_WIFI_SPI_BAUDRATE (8000000)
 
-#define MICROPY_HW_NINA_ACK          pin_find(MP_OBJ_NEW_QSTR(MP_QSTR_ESP_BUSY))
-#define MICROPY_HW_NINA_CS           pin_find(MP_OBJ_NEW_QSTR(MP_QSTR_ESP_CS))
-#define MICROPY_HW_NINA_RESET        pin_find(MP_OBJ_NEW_QSTR(MP_QSTR_ESP_RESET))
-
 // BLE definitions
 #define MICROPY_PY_BLUETOOTH_NINAW10 (1)
 
 #define MICROPY_HW_BLE_UART_ID       (1)
 #define MICROPY_HW_BLE_UART_BAUDRATE (115200)
+#define MICROPY_HW_BLE_UART_FLOW_CONTROL (0)
+
+#if MICROPY_HW_WIFI_AIRLIFT
```

Comment:
I don't understand the Airlift vs non-Airlift build.  It just looks like a difference in pin assignments for the Nina?  Why does there need to be two build variants, why not just always use the on-board ESP32?

Reply:
The intention was to allow using the Airlift shield with the AIrlift-SD board. Then the signals are placed at other pins.